lib/perl/Genome/InstrumentData/AlignmentResult/Command/PicardRnaSeqMetrics.t

```diff
@@ -3,11 +3,53 @@
 use strict;
 use warnings;
 
+use Test::Exception;
+use Test::MockObject;
+use Test::More tests => 2;
+
 use above 'Genome';
-use Test::More tests => 1;
 
-# This test was auto-generated because './InstrumentData/AlignmentResult/Command/PicardRnaSeqMetrics.pm'
-# had no '.t' file beside it.  Please remove this test if you believe it was
-# created unnecessarily.  This is a bare minimum test that just compiles Perl
-# and the UR class.
-use_ok('Genome::InstrumentData::AlignmentResult::Command::PicardRnaSeqMetrics');
+my $class = 'Genome::InstrumentData::AlignmentResult::Command::PicardRnaSeqMetrics';
+use_ok($class) or die;
+
+subtest 'does_annotation_build_have_required_files' => sub{
+    plan tests => 5;
+
+    my $annotation_build = Test::MockObject->new;
+    $annotation_build->set_always('rRNA_MT_file', 'foo');
+    $annotation_build->set_always('annotation_file', 'foo');
```

Comment:
These may cause failures if the person running tests has created a file named `foo` in their working directory.  (This probably won't catch me out, since it doesn't say `turkey` :smile:)

`/dev/null` could be an option (or naming a non-existent file in a temporary directory!)